src/main/python/pre_fx_controller.py

```diff
@@ -211,6 +216,16 @@ def show_qc_criteria(self):
             self
         )
 
+    def export_manual_states_to_json(self):
+        """Prompt the user to select the JSON file for saving manual qc states
+        as well as provenance information
+        """
+
+        path, _ = QFileDialog.getSaveFileName(
+            self, "save JSON file", str(Path.cwd()), "All Files (*);;JSON files (*.json)"
```

Comment:
one thing we need to provide for MPE is the ability to prompt an output path based on a cmdline arg to this app. This would require updating this line to point at the right output as well as adding an ArgumentParser to the main function.

Reply:
I added this functionality, thanks